optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/OptaPlannerProcessor.java

```diff
@@ -551,42 +537,39 @@ private GeneratedGizmoClasses generateDomainAccessors(SolverConfig solverConfig,
             for (DotName dotName : DotNames.GIZMO_MEMBER_ACCESSOR_ANNOTATIONS) {
                 membersToGeneratedAccessorsFor.addAll(indexView.getAnnotations(dotName));
             }
+            membersToGeneratedAccessorsFor.removeIf(this::shouldIgnoreMember);
 
             for (AnnotationInstance annotatedMember : membersToGeneratedAccessorsFor) {
                 switch (annotatedMember.target().kind()) {
                     case FIELD: {
                         FieldInfo fieldInfo = annotatedMember.target().asField();
                         ClassInfo classInfo = fieldInfo.declaringClass();
 
-                        if (!shouldIgnoreMember(classInfo)) {
-                            try {
-                                generatedMemberAccessorsClassNameSet
-                                        .add(GizmoMemberAccessorEntityEnhancer.generateFieldAccessor(annotatedMember, indexView,
-                                                debuggableClassOutput,
-                                                classInfo, fieldInfo, transformers));
-                            } catch (ClassNotFoundException | NoSuchFieldException e) {
-                                throw new IllegalStateException("Fail to generate member accessor for field (" +
-                                        fieldInfo.name() + ") of class " +
-                                        classInfo.name().toString() + ".", e);
-                            }
+                        try {
+                            generatedMemberAccessorsClassNameSet
+                                    .add(GizmoMemberAccessorEntityEnhancer.generateFieldAccessor(annotatedMember, indexView,
+                                            classOutput,
+                                            classInfo, fieldInfo, transformers));
+                        } catch (ClassNotFoundException | NoSuchFieldException e) {
+                            throw new IllegalStateException("Fail to generate member accessor for field (" +
+                                    fieldInfo.name() + ") of class " +
+                                    classInfo.name().toString() + ".", e);
```

Comment:
To make it consistent with the error message at line 571
```suggestion
                                    fieldInfo.name() + ") of the class( " +
                                    classInfo.name().toString() + ").", e);
```